### ISSUE TITLE
Leveled Actor baseref sync

### ISSUE THREAD

**Issue by Dragonissa** [tiltedphoques/TiltedEvolution#639]
## Description

Even with the lastest version (1.6.5), leveled actors (bandits, foxes, bears, etc) can spawn with different baseref on each clients due the fact that its a list of possible monster/animals based on the player level and RNG.

### One potential fix is the following:

OnCellLoad ->

**Case: Party Leader loads the cell after someone else loaded the cell**

Compare loaded leveled actors baseref with the one the party leader wants to spawn. If they match, don't do anything. If they don't, disable existing ones and spawn party leader ones.

**Case: Non Party Leader loads the cell after someone else loaded the cell**

Compare loaded leveled actors baseref with the one the party leader already spawned. If they match, don't do anything. If they don't, disable existing ones and spawn the party leader ones.

## Additional Notes 

﻿**what was it again with Leveled Actors. Level based spawning was synced right?**

yes 
except when it doesnt
if a troll is disabled for level 1 players, but enabled for level 10 players, the troll will appear for both players
if an actor can be a troll or a bear depending on the player's level, there will probably still be desync

but if you're roughly the same level, it shouldn't matter too much
plus, something will appear regardless
so far, i haven't found it to be a big deal
usually, it's just a skeever on one client, and a wolf on another
both take like 3 hits, so its kinda whatever
the health and damage of the entity will still be synced

**so youre fighting a skeever with the health of a wolf?**

basically, yeah
depending on who owns the actor

The host of that actor determines its inventory
Regardless of leveled actor mismatches
This is why you might find skeever tail on what looks like a bear for you

https://www.reddit.com/r/SkyrimTogether/comments/ee9n73/i_see_a_fox_friend_sees_a_bunny/

Potentional quest where this could be a **visual** issue https://elderscrolls.fandom.com/wiki/Animal_Extermination

**Comment by miredirex** [comment]
Also #338
